charts/gardener-extension-shoot-dns-service/templates/dnsman-service-remoteaccess.yaml

```diff
@@ -2,8 +2,9 @@
 apiVersion: v1
 kind: Service
 metadata:
-  {{- if .Values.dnsControllerManager.remoteaccess.service.annotations }}
   annotations:
+    networking.resources.gardener.cloud/from-world-to-ports: '[{"protocol":"TCP","port":{{ .Values.dnsControllerManager.remoteaccess.port }}}]'
```

Comment:
When do we need this? Only when the service type is `LoadBalancer`?

Reply:
Yes, applied condition in [af9bd2f](https://github.com/gardener/gardener-extension-shoot-dns-service/pull/197/commits/af9bd2f5334416b515911616b08d6d2613274cb8).